addons/koji/common/src/main/java/org/commonjava/indy/koji/content/KojiContentManagerDecorator.java

```diff
@@ -426,6 +425,22 @@ private RemoteRepository createRemoteRepository( ArtifactRef artifactRef, final
         }
     }
 
+    private String getRepoName( final KojiBuildInfo build )
+    {
+        StringSearchInterpolator interpolator = new StringSearchInterpolator();
+        interpolator.addValueSource( new ObjectBasedValueSource( build ) );
+
+        try
+        {
+            return interpolator.interpolate( isBinaryBuild( build ) ?
+                    config.getBinayNamingFormat() : config.getNamingFormat() );
+        }
+        catch ( InterpolationException e )
+        {
+            throw new RuntimeException( "Cannot resolve expressions in Koji configuration.", e );
+        }
+    }
+
     private Group adjustTargetGroup( final RemoteRepository buildRepo, final Group group )
```

Comment:
I think we need a new parameter here to tell it whether it's looking for a *binary* target group or a normal target group...